### ISSUE TITLE
Miovision - backfill new intersections

### ISSUE THREAD

**Issue by cczhu** [CityofToronto/bdit_data-sources#416]
Following #402, and possibly another issue that adds the new Miovision locations that have yet to come online as of 2021-06-23, we'll need to backfill station volumes to their activation dates (found in `miovision_api.intersections`).

Tasks:
- [x] Backfill volumes for intersections 58 - 64 from their activation dates to 2021-06-17 inclusive. 2021-06-18 to 2020-06-21 one-minute data already exists in `miovision_api.volumes`.
- [x] Aggregate volumes for intersections 58 - 64 from their activation dates to **2021-06-21** inclusive. 2021-06-18 to 2021-06-21 data was not aggregated to 15-minutes due to the 1-hour offset used in 15-minute aggregation (aggregation starts at 23:00 the previous day, to 22:59 the current day).

Wild card issues:
- We're currently working on bike approach volumes in #403; we'll need to backfill these once they're done.
- We don't properly handle class 7 volumes at many intersections, including the newly added ones (#406). May need to backfill (or at least redo aggregation) for these.
- Once #414 is merged to master it may be possible to manually run the Postgres block below without specifying intersection or classification_uid, since duplicates will be dropped:

```
SELECT miovision_api.find_gaps(%s::date, %s::date) ;
SELECT miovision_api.aggregate_15_min_tmc(%s::date, %s::date) ;
SELECT miovision_api.aggregate_15_min(%s::date, %s::date) ; 
SELECT miovision_api.report_dates(%s::date, %s::date) ;
```

**Comment by radumas** [comment]
Running this little script.
```nohup /etc/airflow/data_scripts/.venv/bin/python3 intersection_tmc.py run-api --path /etc/airflow/data_scripts/volumes/miovision/api/config.cfg --start_date=2020-12-22 --end_date=2021-05-14 --intersection=58 --pull  > 2021-11-09_miovision_backfill_58.log &
nohup /etc/airflow/data_scripts/.venv/bin/python3 intersection_tmc.py run-api --path /etc/airflow/data_scripts/volumes/miovision/api/config.cfg --start_date=2021-05-14 --end_date=2021-06-20 --intersection=58 --intersection=59 --intersection=60 --intersection=61 --intersection=62 --intersection=63 --intersection=64 --pull > 2021-11-09_miovision_backfill_58_64.log &```

**Comment by radumas** [comment]
Need to pull June 22 for all intersections and then aggregate June 21 and 22
